src/github.js

```diff
@@ -2,30 +2,39 @@ const { graphql } = require("@octokit/graphql")
 const { request } = require("@octokit/request")
 const composer = require('./composer')
 const { getHyphenatedDate } = require('./date')
+const docker = require('./docker')
 const mem = require('mem')
 const npm = require('./npm')
 
-const findFile = async (token, repo, fileName, initialPath = '') => {
-  const items = await listRepoContents(token, repo, initialPath)
+const findFile = async (token, repo, fileName) => {
+  const files = await findFiles(token, repo, fileName)
+  if (files.length === 1) {
+    return files[0]
+  } else if (files.length > 1) {
+    const filePaths = files.map(file => file.path)
+    console.error(`Too many matches found: ${JSON.stringify(filePaths, null, 2)}`)
+  }
+  return undefined
+}
+
+const findFiles = async (token, repo, fileName, initialPath = '') => {
+  const items = await listRepoContentsUsingCache(token, repo, initialPath)
   
+  const matchingFiles = []
   const files = items.filter(item => item.type === 'file')
```

Comment:
Does adding `&& item.name === fileName` to the filter work?

Reply:
Also, can there be multiple files with the same name in the same directory?

Reply:
Oh, I suppose adding that to the filter would work, yeah.

And I don't think there can be multiple files with the same name in the same directory.

Reply:
Since this is already a release PR, I'll wait to make that change in a feature branch after this release.